sdk/core/azure-core/test/ut/transport_adapter_base_test.cpp

```diff
@@ -103,6 +111,13 @@ namespace Azure { namespace Core { namespace Test {
     auto expectedResponseBodySize = std::stoull(response->GetHeaders().at("content-length"));
 
     CheckBodyFromBuffer(*response, expectedResponseBodySize);
+
+    json responseJson = json::parse(response->GetBody());
+
+    // The body was 1024 bytes, so the content-length should be 1024 bytes.
```

Comment:
The comment makes sense, but the test isn't asserting this fact. Should we add a length check?
If not, given we are already doing a string comparison anyway, I wonder if there is any value in the comment or if it can be removed.